hnn_core/tests/test_network.py

```diff
@@ -50,34 +50,33 @@ def test_network():
                        match="'times' is an np.ndarray of simulation times"):
         _ = CellResponse(times=[1, 2, 3])
 
-    # Assert that all external feeds are initialized
+    # Assert that all external drives are initialized
     n_evoked_sources = net.n_cells * 3
     n_pois_sources = net.n_cells
     n_gaus_sources = net.n_cells
-    n_common_sources = 2
+    n_comm_sources = 2
 
     # test that expected number of external driving events are created, and
     # make sure the PRNGs are consistent.
-    assert isinstance(net.feed_times, dict)
+    assert isinstance(net.drive_event_times, dict)
     # single trial simulated
-    assert all(len(src_feed_times) == 1 for
-               src_type, src_feed_times in net.feed_times.items()
-               if src_type != 'tonic')
-    assert len(net.feed_times['common'][0]) == n_common_sources
-    assert len(net.feed_times['common'][0][0]) == 40  # 40 spikes
-    assert isinstance(net.feed_times['evprox1'][0][0], list)
-    assert len(net.feed_times['evprox1'][0]) == net.n_cells
-    assert_allclose(net.feed_times['evprox1'][0][0],
+    assert all(len(src_event_times) == 1 for
+               src_type, src_event_times in net.drive_event_times.items())
```

Comment:
There is a very similar test now in `conftest.py`, do we need both?

Reply:
The test in `conftest` is for multiple trials